UNRELEASED.yaml

```diff
@@ -1,5 +1,14 @@
 # UNRELEASED:
 
+BREAKING:
+  - bpk-svgs:
+    - Removed radio svgs elements as these are no longer used.
```

Comment:
If I read this as a consumer of Backpack I wouldn't really know what to make of this information — can you provide some more details on what exactly has been removed and what it means? There was never a React element for these so I think they were only usable from scss.

Something along the lines of "this SVGs that were being used in the radio component have been removed because the radio component no longer uses them. Therefore this change is only breaking if you were using the following Sass variables (or however they were used). Not that exact thing but you get the idea 😅

Reply:
What do you think of the change? Used mostly your sentence as it made sense